tools/addlicense/utils.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func isShebangLine(line string) bool {
+	return strings.HasPrefix(line, "#!")
+}
+
+func closeFile(f *os.File) {
+	err := f.Close()
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+func getLicenseForFile(filePath string) string {
+	switch filepath.Ext(filePath) {
+	case ".go":
+		return goLicense
+	case ".sh":
+		fallthrough
+	case ".py":
+		fallthrough
+	case ".bash":
+		fallthrough
+	case ".zsh":
+		return bashPythonLicense
+	default:
+		return ""
+	}
+}
```

Comment:
Go supports lists in cases, fallthrough is too verbose:
```suggestion
func getLicenseForFile(filePath string) string {
	switch filepath.Ext(filePath) {
	case ".go":
		return goLicense
	case ".py", ".sh", ".bash", ".zsh":
		return bashPythonLicense
	default:
		return ""
	}
}
```